filament/backend/src/vulkan/VulkanBlitter.cpp

```diff
@@ -222,6 +222,10 @@ void VulkanBlitter::lazyInit() noexcept {
     VkShaderModule fragmentShader = decode(VKSHADERS_BLITDEPTHFS_DATA, VKSHADERS_BLITDEPTHFS_SIZE);
     mDepthResolveProgram = new VulkanProgram(mContext, vertexShader, fragmentShader);
 
+    // Allocate one anonymous sampler at slot 0.
+    mDepthResolveProgram->samplerGroupInfo[0].samplers.reserve(1);
+    mDepthResolveProgram->samplerGroupInfo[0].samplers.resize(1);
+
```

Comment:
I'm a little concerned that a change such as the morphing change requires changes in the backend. This should have been a "filament side" change, AFAICT.

Reply:
Oh it actually wasn't the texture morphing change, it was the "split up samplers by stage" change.